src/patternfly/components/AppLauncher/app-launcher-menu-item-description.hbs

```diff
@@ -0,0 +1,6 @@
+<p class="pf-c-app-launcher__menu-item-description{{#if app-launcher-menu-item-description--modifier}} {{app-launcher-menu-item-description--modifier}}{{/if}}"
```

Comment:
Since a `<button>` only accepts [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content), looks like this will need to be a `<span>`.